charmbox-setup.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -ex
+
+# Bundletester and friends in tims ppa
+sudo add-apt-repository -y ppa:tvansteenburgh/ppa
+
+sudo apt-get update -qqy
+sudo apt-get install -qy  \
+                     build-essential \
+                     juju-deployer \
+                     libssl-dev \
+                     make \
+                     python-dev \
+                     python-jujuclient \
+                     python3-dev \
+                     python-pip \
+                     python3-pip \
+                     python-virtualenv \
```

Comment:
Well, we probably dont even need the virtualenv package really. Charmbox/Jujubox is very much intended to be ephemeral. Which kind of negates the need for virtualenvs. I'm sure there are those that would disagree with me.